pnpm-lock.yaml

```diff
@@ -506,7 +506,7 @@ importers:
       web-runtime: link:../web-runtime
     devDependencies:
       '@jest/globals': 29.3.1
-      '@vueuse/core': 9.8.2_vue@3.2.45
+      '@vueuse/core': 9.8.2
```

Comment:
Do you know where this change came from? 🤔 cc @dschmidt

Reply:
probably because of my rebase, vscode had a bad day yesterday and did weird things, totally not my fault :trollface: